### ISSUE TITLE
Remove Debug from Production Build

### ISSUE THREAD

**Issue by vhmth** [siphontv/webm-byte-stream#1]
Right now the `main` script in `package.json` points to `lib/webm-byte-stream` which imports `debug` as a dependencies. `debug` is a dev dependency, so when people try to import `webm-byte-stream` into their projects, node will throw an error saying that that package is missing.

My vote is for having a built file (perhaps in `dist/index.js` or something?) that excludes the `debug` module.

**Comment by eadle** [comment]
Yeah, this has definitely been a problem for me too. Been filtering debug log by prefixing node with DEBUG=...
